README.md

```diff
@@ -22,9 +22,9 @@ and built on top of Apache sling.
 
 ### Prerequisites
 
-* You must have an account on GitHub.  
+* You must have an account on GitHub.
```

Comment:
Thanks, invisible whitespace bugs me. :)

Reply:
Agree!! I just realized my editor hadn't been removing trailing white space for me this week.

Thanks!